src/online2/online-feature-pipeline.h

```diff
@@ -164,9 +164,17 @@ class OnlineFeaturePipeline: public OnlineFeatureInterface {
   virtual int32 NumFramesReady() const;
   virtual void GetFrame(int32 frame, VectorBase<BaseFloat> *feat);
 
+  /// Set the frame offset value. Frame offset is initialized to 0 when the
+  /// feature pipeline is constructed and stays as 0 unless this method is
+  /// called. This method is useful when we want to reset the decoder state,
+  /// i.e. call decoder.InitDecoding(), but we want to keep using the same
+  /// feature pipeline, e.g. in case of an endpoint. The frame offset affects
+  /// the behavior of IsLastFrame, NumFramesReady and GetFrame methods.
```

Comment:
Thanks.  Please use the () when referring to functions in documentation.

Reply:
Done. Also added a check to make sure `0 <= frame_offset`.